internal/pkg/events/incoming/ceilometer.go

```diff
@@ -0,0 +1,152 @@
+package incoming
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+	"time"
+)
+
+//ceilomterGenericIndex value represents ElasticSearch index name for data from which it
+// is not possible to clearly construct indexs name
+const ceilometerGenericIndex = "ceilometer_generic"
+
+var (
+	rexForOsloMessage       = regexp.MustCompile(`"oslo.message"\s*:\s*"({.*})"`)
+	ceilometerAlertSeverity = map[string]string{
+		"audit":    "info",
+		"info":     "info",
+		"warn":     "warning",
+		"warning":  "warning",
+		"critical": "critical",
+		"error":    "critical",
+		"AUDIT":    "info",
+		"INFO":     "info",
+		"WARN":     "warning",
+		"WARNING":  "warning",
+		"CRITICAL": "critical",
+		"ERROR":    "critical",
+	}
+)
+
+type AlertKeySurrogate struct {
```

Comment:
Can you add a comment to this type to make go lint happy?

Reply:
That's next on my list. Just gotta get the kids and do a meeting first.